sources/packages/backend/apps/api/src/route-controllers/application-restriction-bypass/models/application-restriction-bypass.dto.ts

```diff
@@ -1,18 +1,19 @@
 import { RestrictionType } from "@sims/sims-db";
 
 /**
- * Base DTO for application restriction bypass.
+ * DTO class for AEST application restriction bypass summary.
  */
-export class ApplicationRestrictionBypassBaseAPIOutDTO {
+export class ApplicationRestrictionBypassSummary {
   id: number;
   restrictionType: RestrictionType;
   restrictionCode: string;
+  isActive: boolean;
```

Comment:
Should we use isRestrictionActive instead of just isActive to avoid confusion? @dheepak-aot  any suggestion for this?

I am requesting this because it is a part of ApplicationRestrictionBypassSummary. One can confuse it with the isRestrictionBypassActive.

Reply:
I'm good with either `isActive` or `isRestrictionActive`, but `isRestrictionActive` gives better understanding.

Reply:
`isRestrictionActive` sounds good to me.

Reply:
`isActive` is updated to `isRestrictionActive`.